EIPS/eip-2309.md

```diff
@@ -46,6 +45,10 @@ The address executing the transaction **MUST** own all the tokens within the ran
 
 The `fromTokenId` and `toTokenId` **MUST** be a consecutive range of tokens IDs.
 
+The `fromTokenId`, `fromAddress`, and `toAddress` **MUST** be indexed parameters
```

Comment:
Side-comment: Solidity calls them "indexed", but in the protocol they are called "topics". Anyway, since this standard targets mostly Solidity developers, this is the better option.